test/parallel/test-fs-chmod.js

```diff
@@ -62,7 +62,7 @@ function closeSync() {
 }
 
 
-// On Windows chmod is only able to manipulate read-only bit
```

Comment:
It feels pretty strange to talk about the read-only attribute on windows in this context (attempting to convert between the Windows permission model and the unix one) so I just changed this to `write permission`.